os/tools/check_package_size.py

```diff
@@ -113,8 +113,9 @@ def check_binary_header(bin_type):
 COMMON_PART_TMP_SIZE = FLASH_SIZE
 APP1_PART_TMP_SIZE = FLASH_SIZE
 APP2_PART_TMP_SIZE = FLASH_SIZE
-
+size=len(SIZE_LIST)
```

Comment:
Requesting you to follow certain coding guidelines: - 
There should always be a space before and after operators